game/static/game/js/level_editor.js

```diff
@@ -2094,25 +2091,20 @@ ocargo.LevelEditor = function() {
                 }
             }
 
-            if(!connected) {
-                return false;
-            }
+            return !connected && notAlreadyOccupied(sourceNode, controlledNode);
```

Comment:
Previously is connected was false, we returned false. Now, if it's true, we'll return false.